### ISSUE TITLE
Sample sessions - Function error

### ISSUE THREAD

**Issue by btitaro** [bcgov/bccamtrap#26]
Hi Andy - got another error in relation to sample session function: 
Error in **make_sample_sessions(image_data)** : 
  **could not find function "make_sample_sessions**"

Thoughts?

**Comment by ateucher** [comment]
My only thought is that if you're restarted R and/or reinstalled the package you need to call `library(bccamtrap)` again to load it...

**Comment by btitaro** [comment]
Yep I think that might be it. Having trouble successfully removing prior installation of some packages, and installing the overall package. Will work through it and let you know if that solves the problem.

**Comment by ateucher** [comment]
Yeah Windows in particular can be cranky about installing packages that are currently loaded. Best thing is to restart R before installing. If you get really stuck with it, shut down R/Rstudio completely, find the library where packages are installed, and delete the bccamtrap package. Then open RStudio back up and install it again.
